apps/fft/fft-serial.cpp

```diff
@@ -81,7 +139,10 @@ int main(int argc, char* argv[])
     Timer timer;
 
     // fft radix-2 algorithm
-    fft_serial(y_n.data(), N, N);
+    auto &&y = fft_serial(x_n.data(), N, args.debug);
```

Comment:
I wonder is there any reason using r value reference syntax here `auto&&`?  Does it make sense to have `auto const &`?

Reply:
done.